packages/node/src/indexer/indexer.manager.ts

```diff
@@ -198,15 +195,15 @@ export class IndexerManager {
       raw: true,
     });
 
-    const keyValue: KeyValue = {};
+    const keyValue: { [key: string]: string | number | boolean } = {};
 
-    entries.map((entry) => {
+    entries.forEach((entry) => {
       const { key, value } = entry;
       keyValue[key] = value;
     });
 
-    //block offset and genesisHash should only been create once, never update
-    //if offset is changed, will require re-index and re-sync poi.
+    //blockOffset and genesisHash should only been create once, never update
+    //if blockOffset is changed, will require re-index and re-sync poi.
     if (!Object.prototype.hasOwnProperty.call(keyValue, 'blockOffset')) {
```

Comment:
`hasOwnProperty` only check if the property exists, but not the value of it.
You can use `!keyValue.blockOffset` for checking both the property itself and the value